environments/production/known_hosts

```diff
@@ -28,12 +28,12 @@
 10.202.10.238,10.202.10.238 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBFPb3UG0OYM+f1wa1uIa0EyXcrJAUeBRWNupik11oyiZqDoMnlNodI4eoXyRDywNKCnpDB0Cq6k48DJWvr+VZyg=
 10.202.10.238,10.202.10.238 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIL5Ufct69wD1ZIzlt8yEC45kuv6/eqBIbDaQu3Wlr/LR
 10.202.10.238,10.202.10.238 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDAP4W5JFl2/KyWsWahW7+NfL9Z7rZUUfSjxMkeIqwGRTzBR3OXOp055GmU7IL9QrxR1JnhQ6sBD6keZ6UMohU0bnmNjHGsSDp5M3PuvPzP9RxMPQ9SHdMZGIsy4kGhV1V8kNE+gM34NmBvlP9e+fQ1XAICt2BCeAmv33fkHY+9C5SdeYqrLDVtWGBo8Z6oHQsQTTcQHMSNH8KEwPV/zrNo74095AOtrD7Xx65iromxLbC4G0JUgFvNMOvexn+6Q6CfgKQ3siVIUJ/9NhY6dCmlEDZB61cREn/YU7GknceRc4UQJK0KBQn6joZU9tatUhTExQJdwSXpDyH698Wd+1LV
-10.202.10.4, ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBHZvgnRBtcOkDc3GwJOMPsnfhclZZ5L7QApctnjvYn61Xn/hm8Bps0iF7Fcp06HNDD0St6Ka96BT40sXvRcUdb0=
```

Comment:
Any particular reason for this re-ordering?

Reply:
That's just what happened when I ran `cchq production update-local-known-hosts --limit celery6`. At the end of that run, the file is sorted so that there's a standard format (to minimize diffs, ironically in this particular case).

I'm not totally sure how these might have gotten out of order, possibly someone adding 10.202.10.4 in a more manual way